### ISSUE TITLE
Error when ploting in 0.10.0

### ISSUE THREAD

**Issue by Jonghyun-Yun** [igraph/python-igraph#565]
The below throws `NameError: name 'plot' is not defined` in `igraph 0.10.0`. 
```
g = Graph.DataFrame(pd.DataFrame(list(zip(["1", "2", "3"], ["2","1","1"])), columns = ["src", "des"]) , directed=False, use_vids=False)
fig, ax = plt.subplots(figsize=(10,10))
plot(g, target=ax)
```

**Comment by szhorvat** [comment]
Plotting works fine on my end with 0.10.0. To determine what goes wrong on your computer, please show a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by iosonofabio** [comment]
Is matplotlib installed in that environment?

**Comment by szhorvat** [comment]
@iosonofabio This is not related to matplotlib. It happens because when doing `from igraph import *` instead of `import igraph as something`, only those symbols will be imported which are in `__all__`: https://github.com/igraph/python-igraph/blob/master/src/igraph/__init__.py#L1127

**Comment by ntamas** [comment]
Hmmm, yes. We did not have `__all__` in 0.9 or before and therefore everything that the top-level igraph module imported from elsewhere was automatically re-exported. We started sorting this out in 0.10 by specifying an explicit `__all__`, and apparently we missed this :( I'm making a diff of the namespace between 0.9.11 and 0.10.0 to see what we missed and I'll sort it out soon.

**Comment by ntamas** [comment]
Stuff that is missing from `__all__` in 0.10.0:

```
['BoundingBox', 'ClusterColoringPalette', 'Clustering', 'CohesiveBlocks', 'Configuration', 'Cover', 'DefaultGraphDrawer', 'Dendrogram', 'DyadCensus', 'FittedPowerLaw', 'Flow', 'GraphSummary', 'Histogram', 'Matrix', 'Palette', 'Plot', 'Point', 'PrecalculatedPalette', 'RainbowPalette', 'Rectangle', 'RunningMean', 'TriadCensus', 'UniqueIdGenerator', 'VertexClustering', 'VertexCover', 'VertexDendrogram', 'autocurve', 'clustering', 'color_name_to_rgb', 'color_name_to_rgba', 'compare_communities', 'configuration', 'copyfileobj', 'cut', 'datatypes', 'dbl_epsilon', 'defaultdict', 'deprecated', 'drawing', 'formula', 'get_include', 'gzip', 'init_configuration', 'is_graphical', 'known_colors', 'layout', 'load', 'matching', 'math', 'mean', 'median', 'multidict', 'name', 'named_temporary_file', 'numpy_to_contiguous_memoryview', 'operator', 'operators', 'os', 'palettes', 'percentile', 'plot', 'power_law_fit', 'quantile', 'read', 'rescale', 'safemax', 'safemin', 'save', 'sparse_matrix', 'split_join_distance', 'statistics', 'summary', 'sys', 'utils', 'version', 'warn', 'write']
```

Some of these are indeed not needed (like `sys` or `math`, which are simply modules that the old 0.9.11 igraph module imported for its own purposes), while others should be exported (like `plot` or `compare_communities`).

**Comment by szhorvat** [comment]
@ntamas If you are planning a 0.10.1, consider doing one with the C core at the same time.

**Comment by iosonofabio** [comment]
I'd be in favour of releasing a 10.0.1 independent of the C core, since it's not API breaking except for bugfixes

**Comment by ntamas** [comment]
There are pending fixes in the 0.10 branch of the C core and I guess @szhorvat wants to get those out as well if we are making a new release anyways.

**Comment by iosonofabio** [comment]
Got it, thanks both

**Comment by ntamas** [comment]
Fixed in e99e084
